src/qt/locale/dynamic_sv.ts

```diff
@@ -5543,8 +5543,8 @@ till exempel: alertnotify=echo %%s | mail -s &quot;Dynamic Alert&quot; admin@foo
         <translation>BDAP Länkdetalj</translation>
     </message>
     <message>
-        <source>Requestor:</source>
-        <translation>Begäran:</translation>
+        <source>Requetor:</source>
```

Comment:
Shouldn't the English source be "Requester"?

Reply:
Good catchhhhhh